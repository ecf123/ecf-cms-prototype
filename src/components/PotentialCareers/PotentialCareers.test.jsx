import { render, screen } from "@testing-library/react";
import Careers from "./PotentialCareers.jsx";
import { customRender } from "../../utils/testUtils";

it("it should match the snapshot", () => {
    const { container } = customRender(
      <Careers/>
    );
    expect(container).toMatchSnapshot();
  });

it("Should render in the potential careers heading", () => {
  // 1. Arrange
  render(<Careers />);
  // 2. Act
  const heading = screen.getByText("Potential Careers");
  // 3. Assert
  expect(heading).toBeInTheDocument();
  //comment
});

it("Should render in the potential careers list", () => {

  render(<Careers />);

  const listItems = screen.queryAllByText("Career");
  
  expect(listItems).toBeTruthy();
});
