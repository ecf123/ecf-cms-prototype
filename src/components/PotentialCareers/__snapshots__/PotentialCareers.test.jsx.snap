// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should match the snapshot with props 1`] = `
<div>
  <div
    class="career"
  >
    <h5
      class="career__heading"
    >
      Potential Careers
    </h5>
    <ul
      class="career__list"
    >
      <li
        class="career__item"
      >
        job1
      </li>
      <li
        class="career__item"
      >
        job2
      </li>
      <li
        class="career__item"
      >
        bigjob
      </li>
    </ul>
  </div>
</div>
`;
